_news/2019-10-11-forseti-and-terraform-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Exciting Changes to Forseti and Terraform in 2.23
+author: Ken Evensen and ...
+---
+
+Forseti Community,
+
+With Forseti 2.23 come some exciting changes.  
+
+### Python Installer Deprecation
+As we outlined in our previous [post]({% link _news/2019-09-17-deprecate-python-installer.md %}), we have deprecated the Python based installer.  Additionaly, we have removed the related Python installer code from the forseti-security repo. Forseti now supports Terraform as the official installation path.
+
+We have created a migration script and documentation to help you seamlessly 
+migrate from Deployment Manager to Terraform:
+
+* For v2.18.0+ users, refer to the instructions to migrate 
```

Comment:
Is it necessary to mention the instructions going from 4.3 --> 5.0?

Reply:
My impression is that this text as is is still applicable to going from 4.3 to 5.0.  But I am open to refactoring the text if more clarity is needed.